package.json

```diff
@@ -27,12 +26,16 @@
     "docusaurus-plugin-sass": "^0.2.1",
     "file-loader": "^6.2.0",
     "mdx-mermaid": "^1.2.2",
-    "mermaid": "^8.14.0",
+    "mermaid":"8.11.0",
     "prism-react-renderer": "^1.2.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "sass": "^1.37.5",
-    "url-loader": "^4.1.1"
+    "url-loader": "^4.1.1",
+    "webpack":"^5.0.0",
+    "typescript":">= 2.7",
```

Comment:
> warning "@docusaurus/core > react-dev-utils > fork-ts-checker-webpack-plugin@6.5.2" has unmet peer dependency "typescript@>= 2.7".

I don't know what's the use of A, I added it because of the warning. also`webpack`,`@types/react` and `unist-util-visit`.
I tried to delete it, still able to build successfully, If it's really useless, I'll delete it on the next commit.

Reply:
@Selflocking we prefer tolerating warnings than introducing a large dependency. Plz delete it.